.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+<!-- This form is for other issue types specific to WooCommerce E2E Tests Boilerplate. This is not a support portal. -->
```

Comment:
Minor change here to that this for Smooth Generator not for WooCommerce E2E Tests Boilerplate:

```suggestion
<!-- This form is for other issue types specific to WooCommerce Smooth Generator. This is not a support portal. -->
```